scripts/_helpers.py

```diff
@@ -391,6 +391,38 @@ def _get_country(target, **keys):
         return np.nan
 
 
+def _getContinent(code):
+    """
+    Returns continent names that contains list of iso-code countries
+
+    Parameters
+    ----------
+    code : str
+        List of two letter country ISO codes
+
+    Returns
+    -------
+    continent_list : str
+        List of continent names
+
+    Example
+    -------
+    from helpers import _getContinent
+    code = ["DE", "GB", "NG", "ZA"]
+    _getContinent(code)
+    >>> ["africa", "europe"]
+    """
+    from config_osm_data import world_iso
+
+    continent_list = []
+    code_set = set(code)
+    for continent in world_iso:
+        single_continent_set = set(world_iso[continent])
+        if code_set.intersection(single_continent_set):
+            continent_list.append(continent)
+    return continent_list
```

Comment:
**Should be addressed**

I recommend you throw an exception in case the country is not in the list of continents, i.e. in case no match is found.

This code could work (haven't tested it)

```suggestion
        matching_countries = code_set.intersection(single_continent_set)
        if matching_countries:
            continent_list.append(continent)
            code_set = code_set - matching_countries
    
    if code_set:
        raise KeyError(f"Unable to map countries '{code_set}' to a demand data region.")
    return continent_list
```

Reply:
I am not sure about this. Which country list do you want to compare?
If the country is not listed it already leads to an error such as load for "X" not found.
I think the exception is maybe not necessary?